spring-data-relational/src/test/java/org/springframework/data/ProxyImageNameSubstitutor.java

```diff
@@ -0,0 +1,83 @@
+/*
+ * Copyright 2015-2023 the original author or authors.
```

Comment:
Nit: We start with the year of introduction, so Copyright would be 2023 only.

Reply:
Done.